webapp/components/form/AutocompleteDialog/store/useLocalState.js

```diff
@@ -47,7 +55,7 @@ export const useAutocompleteDialog = ({
 
       window.removeEventListener('click', clickListener)
     }
-  }, [State.getInputField(state), list.current])
+  }, [list.current, State.getItems(state), State.getInputField(state)])
```

Comment:
why not to add list to state as well? I saw you pass this around some action.
what do you think?